test/integration/full/preload-cssom/preload-cssom.js

```diff
@@ -0,0 +1,202 @@
+/* global axe, Promise */
+describe('preload cssom integration test pass', function() {
+	'use strict';
+
+	var origAxios;
+
+	before(function(done) {
+		function start() {
+			// Stop messing with my tests Mocha!
+			document.querySelector('#mocha h1').outerHTML =
+				'<h2>preload cssom integration test</h2>';
+			// cache original axios object
+			if (axe.imports.axios) {
+				origAxios = axe.imports.axios;
+			}
+			// run axe
+			axe.run(
+				{
+					preload: true
+				},
+				function(err) {
+					assert.isNull(err);
+					done();
+				}
+			);
+		}
+		if (document.readyState !== 'complete') {
+			window.addEventListener('load', start);
+		} else {
+			start();
+		}
+	});
+
+	function createStub(shouldReject) {
```

Comment:
What happened to `sinon`? It's **much** more feature complete than this.

Reply:
Was not embracing any of the features of sinon :(. Also sinon stub did not work in selenium webdriver. Needed a million hacks to get a stub in place. Until we change to a more modern test suite, this should do the trick.

Reply:
Fair enough. Might be useful to document that somewhere for the "next guy".

Reply:
Sorted!